zebra-consensus/src/transaction/tests.rs

```diff
@@ -807,6 +807,110 @@ async fn mempool_request_with_immature_spend_is_rejected() {
     );
 }
 
+/// Tests that calls to the transaction verifier with a mempool request that spends
+/// mature coinbase outputs to transparent outputs will return Ok() on Regtest.
+#[tokio::test]
+async fn mempool_request_with_transparent_coinbase_spend_is_accepted_on_regtest() {
+    let _init_guard = zebra_test::init();
+
+    let network = Network::new_regtest(None, Some(1_000));
+    let mut state: MockService<_, _, _, _> = MockService::build().for_prop_tests();
```

Comment:
We can use a mocked service for unit tests and remove the `expect`s in the code below.

```suggestion
    let mut state: MockService<_, _, _, _> = MockService::build().for_unit_tests();
```